micall/drivers/sample.py

```diff
@@ -19,10 +19,31 @@
 from micall.core.denovo import denovo
 from micall.g2p.fastq_g2p import fastq_g2p, DEFAULT_MIN_COUNT, MIN_VALID, MIN_VALID_PERCENT
 from micall.utils.driver_utils import makedirs
+from contextlib import contextmanager
 
 logger = logging.getLogger(__name__)
 
 
+@contextmanager
+def open_files(**filemode):
```

Comment:
Please add a docstring to show how to call this. If you don't currently need to handle already open files, then don't bother. Maybe make a comment that we could add support for already opened files in the future.

Reply:
Done!